internal/builtinfunctions/functions.go

```diff
@@ -550,13 +550,11 @@ func getBindConstantsFunction() schema.CallableFunction {
 		schema.NewDisplayValue(
 			schema.PointerTo("Bind Constants"),
 			schema.PointerTo(
-				"Returns a list of objects each containing two properties: an item property which contains "+
-					"the corresponding item from the list in the first parameter; and, a constant property which "+
-					"contains the value of the second input parameter. "+
-					fmt.Sprintf("The two names are joined with the delimiter %q. ", CombinedObjIDDelimiter)+
-					fmt.Sprintf("Each object has two properties `%s` and `%s`.\n", CombinedObjPropertyItemName, CombinedObjPropertyConstantName)+
+				"Returns a list of objects each containing two properties. "+
 					fmt.Sprintf("Param 1: Value(s) to be included in the `%s` field \n", CombinedObjPropertyItemName)+
-					fmt.Sprintf("Param 2: Value(s) to populate the field `%s` with every output item", CombinedObjPropertyConstantName)),
+					fmt.Sprintf("Param 2: Value(s) to populate the field `%s` with every output item\n", CombinedObjPropertyConstantName)+
+					"The output list items will have an Object ID and schema name as documented in "+
+					"https://arcalot.io/arcaflow/workflows/schemas/#generated-combined-schema-names."),
```

Comment:
What we have now seems generally good.  But, if you're inclined to make another pass, I would try to reincorporate some of the previous edit.  E.g., something like
> Returns a list of objects each containing two properties.  The `%s` property contains the corresponding item from the list in the first parameter, and the `%s` property contains the value of the second input parameter.  The returned value will have an Object ID and schema name as documented in https://arcalot.io/arcaflow/workflows/schemas/#generated-combined-schema-names.

Reply:
made the change